lstchain/mc/sensitivity.py

```diff
@@ -1023,8 +1020,6 @@ def sensitivity_gamma_efficiency_real_data(dl2_file_on, dl2_file_off,
 
 
 
-        events_on_after_g_cut=events_bin_on[events_bin_on.gammaness > best_g_cut]
```

Comment:
maybe we could leave these two lines since they are referred to in the next lines.

Reply:
they are unsued because they are commented out in the next line:
```python
best_theta2_cut = tcut[i]#find_cut_real(events_on_after_g_cut, events_off_after_g_cut, obstime_on, obstime_off, "theta2", 0.0, 1.0, gamma_eff_theta2) * u.deg**2
```
in any case, removing them is fine with me